libcxx/.clang-tidy

```diff
@@ -56,8 +56,12 @@ CheckOptions:
     value: _
   - key:   readability-identifier-naming.TemplateParameterIgnoredRegexp
     value: (.*\:auto|expr-type) # This is https://llvm.org/PR56464
-  - key:   readability-identifier-naming.ValueTemplateParameterIgnoredRegexp # TODO: enforce naming of variable parameters
-    value: .*
+  - key:   readability-identifier-naming.ValueTemplateParameterCase
+    value: lower_case
```

Comment:
I really dislike this. I really like to stay close to the wording in the Standard an there value arguments are upper case too,